src/dotnet/APIView/APIViewUnitTests/CrossLanguageMetadataTests.cs

```diff
@@ -0,0 +1,118 @@
+using System.Text.Json;
+using APIView.Model;
+using FluentAssertions;
+using Xunit;
+
+namespace APIViewUnitTests;
+
+public class CrossLanguageMetadataTests
```

Comment:
would be good to add one more test that doesn't include CrossLanguageMetadata at all in the token file, since it's optional